README.md

```diff
@@ -1,65 +1,206 @@
 
-    DEMO
-    
-    ```sh
-    tidiff 'create database demo charset utf8mb4 collate utf8mb4_general_ci'
-    tidiff 'create table demo.tt10000 (a bigint(20) not null auto_increment primary key, b varchar(20) not null, c bigint(20))'
-    # OR
-    # tidiff --mysql.db demo --tidb.db demo 'create table tt10000 (a bigint(20) not null auto_increment primary key, b varchar(20) not null, c bigint(20))'
-    tidiff '! {{$count:=count 10000}} insert into demo.tt10000 values {{range $index := $count}} (NULL, "{{varchar 20}}", "{{int 100000 100000000}}"){{if head $index $count}},{{end}}{{end}}'
-    tidiff 'select count(*) as count, sum(c) from demo.tt10000'
-    tidiff 'select * from demo.tt10000 order by rand() limit 5'
-    ```
+- Generating the same extra workload for both MySQL and TiDB when it comes to the case that a large amount of data (e.g. 1 million rows of data) is needed to reproduce the BUG.
 
-    The results of above commands
-    
-    ![](media/tidiff-guide-demo1.png)
 
-    You can use the command line mode as downstream pipeline, e.g: `randgen | xargs tididff`. The SQL statement should be quote with `'` instead of `"`.
+## Sample usages
+
+The examples assume that you have already started the local MySQL and TiDB servers. The servers can be logged in with root username and an empty password. 
+
+```
+ ~ tidiff -h
+NAME:
+tidiff - Execute SQL in TiDB and MySQL and returns the results
+
+USAGE:
+tidiff [global options] command [command options] [arguments...]
+
+VERSION:
+0.0.1
+
+DESCRIPTION:
+Used to compare the result different in MySQL and TiDB for the same SQL statement
+
+COMMANDS:
+    help, h  Shows a list of commands or help for one command
+
+GLOBAL OPTIONS:
+--mysql.host value      MySQL host (default: "127.0.0.1")
+--mysql.port value      MySQL port (default: 3306)
+--mysql.user value      MySQL username (default: "root")
+--mysql.password value  MySQL password
+--mysql.db value        MySQL database
+--mysql.options value   MySQL DSN options (default: "charset=utf8mb4")
+--tidb.host value       TiDB host (default: "127.0.0.1")
+--tidb.port value       TiDB port (default: 4000)
+--tidb.user value       TiDB username (default: "root")
+--tidb.password value   TiDB password
+--tidb.db value         TiDB database
+--tidb.options value    TiDB DSN options (default: "charset=utf8mb4")
+--log.diff value        Log all query diff to file
+--help, -h              show help (default: false)
+--version, -v           print the version (default: false)
+```
+
+- Create a demo database for MySQL/TiDB
+
+  `tidiff 'create database demo charset utf8mb4 collate utf8mb4_general_ci'`
+
+- Create a demo.tt10000 table for MySQL/TiDB
+
+   `tidiff 'create table demo.tt10000 (a bigint(20) not null auto_increment primary key, b varchar(20) not null, c bigint(20))'`
+
+   or
+   
+   `tidiff --mysql.db demo --tidb.db demo 'create table tt10000 (a bigint(20) not null auto_increment primary key, b varchar(20) not null, c bigint(20))'`
```

Comment:
```suggestion
    ```
    tidiff --mysql.db demo --tidb.db demo 'create table tt10000 (a bigint(20) not null auto_increment primary key, b varchar(20) not null, c bigint(20))'
    ```
```